tuxemon/states/world/worldstate.py

```diff
@@ -262,6 +263,13 @@ def set_transition_surface(self, color: ColorLike) -> None:
         )
         self.transition_surface.fill(color)
 
+    def set_layer(self) -> None:
+        self.layer = pygame.Surface(
```

Comment:
it would be better if this Surface were not created/destroyed each frame.  holding a reference to it by something simple like making it an instance attribute would be better.  pygame surface allocations can be slow, since its all software rendering.